docs/building_lbann.rst

```diff
@@ -137,12 +137,12 @@ CMake flags known to LBANN's "Superbuild" build system.
     :code:`-e` flag. A full list of options can be viewed with the
```

Comment:
Shouldn't it say that the -e option allows you to specify the name of your custom environment?

Reply:
It is. The diff doesn't show line 136, but that's what that is.

Reply:
Ok, I see that, but when you actually try looking at the full list of options using the script it doesn't explain -e. I actually discovered that when I was going through quick_start.rst

Script help menu:
  --help               Display this help message and exit.
  --verbose            Verbose output.
  -d | -deps-only)     Only install the lbann dependencies
  -e | --env           Build and install LBANN in the spack environment provided.
  --half               Enable support for HALF precision data types in Hydrogen and DiHydrogen
  --disable-gpus       Disable GPUS
  -s | --superbuild    Superbuild LBANN with dihydrogen, hydrogen, and aluminum